Symfony/CS/Fixer.php

```diff
@@ -121,6 +136,9 @@ public function addConfig(ConfigInterface $config)
         $this->configs[] = $config;
     }
 
+    /**
+     * @return ConfigInterface[]
```

Comment:
Improving phpdoc is unrelated to adding integration tests. You should submit it as a separate PR